winpr/include/winpr/io.h

```diff
@@ -187,6 +187,37 @@ WINPR_API VOID _IoDeleteDeviceEx(PDEVICE_OBJECT_EX DeviceObject);
 
 #endif
 
+#ifdef _UWP
```

Comment:
UWP is quite different from Win32 and non-Win32, so it is sometimes easier to consider it separately like this.